components/aws/sagemaker/codebuild/scripts/deploy.sh

```diff
@@ -62,7 +66,14 @@ echo "Deploying version ${FULL_VERSION_TAG}"
 
 # Build the image
 FULL_VERSION_IMAGE="${REMOTE_REPOSITORY}:${FULL_VERSION_TAG}"
-docker build . -f Dockerfile -t "${FULL_VERSION_IMAGE}"
+
+if [ "${BUILD_VERSION}" == "v2" ]; then
+    echo "Building V2 image"
+	docker build . -f v2.Dockerfile -t "${FULL_VERSION_IMAGE}"
```

Comment:
nit: indentation here and in the else statement

Reply:
some weird whitespace vs tab issue should be fixed now